src/main/java/com/fruits/ecommerce/services/Utils/EmailService.java

```diff
@@ -1,163 +1,271 @@
+                        <p>Dear %s,</p>
+                        
+                        <div style='
+                            background-color: #f0faf0;
+                            padding: 15px;
+                            margin: 20px 0;
+                            border-radius: 5px;
+                            border-left: 4px solid #27ae60;
+                        '>
+                            <p style='margin: 0;'>Your account has been successfully unlocked. You can now log in.</p>
+                        </div>
+                        
+                        <p>If you have any questions, our support team is here to help.</p>
+                        
+                        <p style='margin-top: 20px;'>
+                            Best regards,<br>
+                            Support Team
+                        </p>
+                    </div>
+                """, firstName);
+
+        sendHtmlEmail(email, "Your Account is Now Unlocked 🔓", htmlContent);
     }
 
-    public void sendPasswordResetToEmail(String firstName, String username, String password, String email)
+    public void sendPasswordResetEmail(String firstName, String username, String password, String email)
             throws MessagingException {
-        SimpleMailMessage message = new SimpleMailMessage();
-        message.setTo(email);
-        message.setFrom(fromEmail);
-        message.setSubject("Password Reset Notification");
-        message.setText(
-                "Password Reset Notification\n" +
-                        "Dear " + firstName + ",\n" +
-                        "We received a request to reset your password for your account." +
-                        "\nYour account details have been updated with the following information:\n" +
-                        "----------------------------------------------------------\n" +
-                        " ** Username: " + username + "\n" +
-                        " ** Email Address: " + email + "\n" +
-                        " ** Your New Password: ( " + password + " )\n" +
-                        "----------------------------------------------------------\n" +
-                        "\nFor your account security:\n" +
-                        "1. Please change this temporary password immediately after logging in\n" +
-                        "2. Never share your password with anyone\n" +
-                        "3. Make sure to use a strong password\n" +
-                        "\nIf you did not request this password reset, please contact our support team immediately.\n" +
-                        "\nNeed help? Our support team is available 24/7.\n" +
-                        "Best regards,\n" +
-                        "ZAG Electronics Industries Corporation\n" +
-                        "www.our-company.com\n" +
-                        "Phone: +201012345678\n" +
-                        "Customer Support Team"
-        );
+        String htmlContent = String.format("""
```

Comment:
You can use template engine like mustache instead of writing HTML inside java, please check https://github.com/spullara/mustache.java